includes/wf_crm_webform_preprocess.inc

```diff
@@ -214,6 +211,9 @@ class wf_crm_webform_preprocess extends wf_crm_webform_base {
         // RM: This is needed in order for CiviCRM to know that this is a 'test' (i.e. 'preview' action in CiviCRM) transaction - otherwise, CiviCRM defaults to 'live' and returns the payment form with public key for the live payment processor!
         $contributionCallbackQuery['action'] = CRM_Core_Action::description(CRM_Core_Action::PREVIEW);
       }
+      if ((strpos($contributionCallbackUrl, '/') !== 0)) {
```

Comment:
A nit, but this is a weird way to check this. Why not just:

```
if ($contributionCallbackUrl[0] != '/') {
```

Reply:
That's a better idea (I copied core.) But i fixed the actual callback URL to have a `/`.